changelog/fragments/bump-kb.yaml

```
@@ -0,0 +1,25 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Bump kubebuilder to the commit which uses controller-runtime v0.8.3. This results
+      in bumping kubernetes dependencies in projects (go/v3) to 1.20.
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "change"
+
+    # Is this a breaking change?
+    breaking: false
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: Scaffold projects with k8s 1.20 dependencies.
```

Only Go projects need to change something
```suggestion
      header: (go/v3) Bump controller-runtime to v0.8.3 and kubernetes dependencies to v0.20.2
```